src/components/ItemList/ItemList.tsx

```diff
@@ -7,21 +7,27 @@ import './ItemList.scss';
 export type Props<T extends { id: string }> = {
     activeItemIndex?: number;
     autoScroll?: boolean;
+    fileId?: string | null;
+    isCurrentFileVersion?: boolean;
     itemRowAs?: React.ElementType;
     items: T[];
     onActivate?: (index: number) => void;
     onSelect: (index: number, event: React.SyntheticEvent) => void;
 };
 
-const ItemList = <T extends { id: string }>({
-    activeItemIndex = 0,
-    autoScroll = false,
-    itemRowAs: ItemRowComponent = ItemRow,
-    items,
-    onActivate = noop,
-    onSelect,
-    ...rest
-}: Props<T>): JSX.Element => {
+const ItemList = <T extends { id: string }>(props: Props<T>): JSX.Element => {
```

Comment:
This file is meant to be a generic component, right? Should the resin tags live on a decorated version of ItemRow, instead?